taf/developer_tool.py

```diff
@@ -466,11 +469,13 @@ def _initialize_roles_and_keystore(roles_key_infos, keystore, enter_info=True):
     that json
     """
     roles_key_infos_dict = read_input_dict(roles_key_infos)
+    if keystore is None:
+        keystore = roles_key_infos_dict.get("keystore", _default_keystore_path())
```

Comment:
When `keystore` is specified in `roles_key_infos` (in configuration json file), that path will be used, but the line saying that the default location will be used is still printed